### PR TITLE
Roy recipe

### PR DESCRIPTION
**Pull request by Rc93s**
i change to English

### CODE REVIEW

**Review by tleivy — CHANGES_REQUESTED** [review]
Also add an entry to index

**Review by tleivy — APPROVED** [review]
很好！

File: recipe/index.md [hunk 1]
```diff
@@ -2,6 +2,7 @@ Recipes
 
 # dessert
     - Flan: dessert made out of tastiness short explanation here
+    - Chocolate Cake
```
Comment (dp25) [inline recipe/index.md:6]: Hmmm , Sounds good

**Review by dp25 — APPROVED** [review]
very nice roy!

**Pull request merged**